warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/EventMapper.java

```diff
@@ -546,10 +484,60 @@ public void map(K1 key, V1 value, Context context) throws IOException, Interrupt
             context.progress();
         }
         
+        incrementEventCount(value, context);
+        
+        updateMetrics(value, eventMapperTimer, fields);
+    }
+    
+    private void failJobOnConstraintViolations(Exception e) {
+        if (e instanceof ConstraintChecker.ConstraintViolationException) {
+            throw ((RuntimeException) e);
```

Comment:
Not your code but can this cast be dropped? Doesn't seem like it's doing anything.